docs/modules/httpd.md

```diff
@@ -76,7 +76,7 @@ configured.
 httpd.start({
   webroot = "<static file prefix>",
   max_handlers = 20,
-  auto_index = httpd.INDEX_NONE || httpd.INDEX_ROOT || httpd.INDEX_ALL,
+  auto_index = httpd.INDEX_ALL,
```

Comment:
This removes the mention of the other index modes, I can't say I'm in favour of that 🙃

Reply:
It turns out that these are actually enums and shouldn't be or'ed together. Worse, the example uses || which is the logical or. Thus, this expression returns the first non-zero value!!

Reply:
Yes, logical OR, because you need to choose one of them. What's the correct syntax for that in this case, if not that? (genuine question, not a snarky comment)